examples/plot_expectation_maximization_ball.py

```diff
@@ -14,7 +14,8 @@
 
 import geomstats.backend as gs
 from geomstats.geometry.poincare_ball import PoincareBall
-from geomstats.learning.expectation_maximization import RiemannianEM
+from geomstats.learning.expectation_maximization import RiemannianEM, \
+    weighted_gmm_pdf
```

Comment:
Should `weighted_gmm_pdf` go in the `sampling` module?

Reply:
It does not sample points from a distribution but evaluates the density function of given points. If the sampling module intends to follow tensorflow/pytorch distribution modules then yes